examples/access-control-directives.ts

```diff
@@ -58,7 +65,10 @@ const schema = makeExecutableSchema({
       throwIfMissingPermissions: (): number => 123,
     },
   },
-  schemaDirectives: { auth, hasPermissions },
+  schemaDirectives: schemaDirectives as Record<
+    string,
+    typeof SchemaDirectiveVisitor
```

Comment:
I saw all of these casts... but to me it's not clear why this is needed.

Always put the tsc failure in the commit message (text, not screenshots ;-)). But please sync with me in our slack, so I can try to help